src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -237,21 +237,39 @@ internal static OtlpCommon.KeyValue ToOtlpAttribute(this KeyValuePair<string, ob
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static OtlpTrace.Status ToOtlpStatus(ref TagEnumerationState otlpTags)
+        private static OtlpTrace.Status ToOtlpStatus(this Activity activity, ref TagEnumerationState otlpTags)
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3103 The current code populates the deprecated field, which is not done in the new path. I am trying to remove that from existing path itself, to keep this PR less complex.

Reply:
Thank you.